build/terraform/template/aws/cluster.tf.tpl

```diff
@@ -16,8 +21,8 @@
 
 // Sifchain terraform module
 module sifchain {
-    source                  = "../../build/terraform/providers/aws"
-    region                  = "us-west-1"
+    source                  = "github.com/sifchain/sifnode/build/terraform/providers/aws"
```

Comment:
I would keep this relative as this means you have to have the repo cloned and you can also make edits locally and not change this unnecessarily